.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: required
+dist: trusty
+language: node_js
+node_js: "8"
+script:
+  - npm test
+services:
+  - postgresql
+before_script:
+  - createuser chuks -s -U postgres
+  - createdb postit-test --username=chuks
+after_success: 
+  - npm run coverage
+  - npm run coveralls
+notifications:
+  - webhooks: https://coveralls.io/webhook?repo_token=yEvDWmpY7RvxsiiRUxGYcb85eNoYYwTac
```

Comment:
### Nitpick

Check your files and make sure you add an extra line to everything so you don't keep seeing this red warning.

### Critical
You just exposed your `repo-token`, you should look into saving in an environment variable instead and not expose it.